src/java/org/apache/cassandra/config/Config.java

```diff
@@ -526,6 +526,13 @@ public class Config
      */
     public volatile int validation_preview_purge_head_start_in_sec = 60 * 60;
 
+    /*
+     * Toggles to turn on the logging or rejection of operations for token ranges that the node does not own,
+     * or is not about to acquire.
+     */
+    public volatile boolean log_out_of_token_range_requests = true;
+    public volatile boolean reject_out_of_token_range_requests = true;
```

Comment:
There's a question of whether rejecting by default is the right way to go about this in a minor release, but rejecting might actually be the safest option, given potential correctness implications for simply logging.

Reply:
reject for this type of error is far better than logging... *knowing* to scan the logs and detect it and how to "fix" are going to be a shocker to operators... so rejecting keeps the DB consistent